.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
```

Comment:
This will miss https://github.com/react-monaco-editor/react-monaco-editor/blob/master/example/package.json. Please add support for it as well.

Reply:
I am not sure how dependabot handles multiple package.json files in a repo after reading a few articles on Google. added a new subsection of `npm` with directory `/example` below. not sure if that works, just my guess.

Reply:
That's my guess as well.